private/bufpkg/bufapiclient/bufapiclient.go

```diff
@@ -67,6 +87,14 @@ func RegistryProviderWithAddressMapper(addressMapper func(string) string) Regist
 	}
 }
 
+// RegistryProviderWithScheme returns a new RegistryProviderOption that adds the given scheme to the transport address
+// for all clients returned by a provider
+func RegistryProviderWithScheme(scheme string) RegistryProviderOption {
```

Comment:
Is this still needed now that you have `PrependHTTPS`?

Reply:
Ugh no.  Removed along with the other scheme options.